front/app/modules/commercial/custom_idea_statuses/admin/components/DeleteStatusButton.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+
+import {
+  Tooltip,
+  TooltipProps,
+  colors,
+} from '@citizenlab/cl2-component-library';
+
+import useDeleteIdeaStatus from 'api/idea_statuses/useDeleteIdeaStatus';
+
+import Button from 'components/UI/Button';
+
+import { FormattedMessage } from 'utils/cl-intl';
+
+import messages from './messages';
+
+interface Props {
+  tooltipDisabled: TooltipProps['disabled'];
+  buttonDisabled: boolean;
+  tooltipContent: TooltipProps['content'];
+  ideaStatusId: string;
+}
+
+const DeleteStatusButton = ({
+  tooltipDisabled,
+  buttonDisabled,
+  tooltipContent,
+  ideaStatusId,
+}: Props) => {
+  const { mutate: deleteIdeaStatus } = useDeleteIdeaStatus();
+
+  return (
+    <Tooltip
+      placement="top"
+      theme="light"
+      disabled={tooltipDisabled}
+      content={tooltipContent}
+      trigger="mouseenter"
+    >
+      <Button
+        onClick={() => deleteIdeaStatus(ideaStatusId)}
+        buttonStyle="text"
+        disabled={buttonDisabled}
```

Comment:
Are the button disabled and tooltip disabled props related? Perhaps we could have a single prop for this?

Reply:
I also found it to be clearer. I'm often confused by Tooltip's disabled value (i.e. that it means "tooltip is disabled").

Reply:
I actually could make it one prop, because tooltip disabled is indeed just the negation of button disabled.

Reply:
Solved it with a compromise: https://github.com/CitizenLabDotCo/citizenlab/pull/8234/commits/57e229f378ea2b517a02152a0f1d2f8fd7857425

No unnecessary prop, but an extra var for clarity inside the buttons.